be/src/agent/agent_task.cpp

```diff
@@ -700,8 +700,9 @@ void run_update_meta_info_task(const std::shared_ptr<UpdateTabletMetaInfoAgentTa
             case TTabletMetaType::DISABLE_BINLOG:
                 break;
             case TTabletMetaType::BINLOG_CONFIG:
+                LOG(INFO) << "update tablet:" << tablet->tablet_id()
```

Comment:
update binlog config of tablet from xx to xx

Reply:
ok, the binlog config after modified will be print in tablet_meta#set_binlog_config